From 8b951a647654e51cabd795532e1504b2ce0880de Mon Sep 17 00:00:00 2001
From: Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
Date: Mon, 3 Feb 2025 10:36:31 -0300
Subject: [PATCH] notification: Pass GUnixFDList on x-d-p >= 1.19.1

This is needed as the impl interface is now marked with
org.gtk.GDBus.C.UnixFD.
---
 src/notification.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/notification.c b/src/notification.c
index b109456..4a1a768 100644
--- a/src/notification.c
+++ b/src/notification.c
@@ -114,6 +114,9 @@ activate_action (GDBusConnection *connection,
 static gboolean
 handle_add_notification (XdpImplNotification *object,
                          GDBusMethodInvocation *invocation,
+#ifdef HAVE_XDP_1_19_1
+                         GUnixFDList *fds G_GNUC_UNUSED,
+#endif
                          const gchar *arg_app_id,
                          const gchar *arg_id,
                          GVariant *arg_notification)
